front/app/containers/Admin/projects/all/new/Projects/Filters/messages.ts

```diff
@@ -89,4 +89,24 @@ export default defineMessages({
     id: 'app.containers.Admin.projects.all.new.Projects.Filters.Folders.folders',
     defaultMessage: 'Folders',
   },
+  visibilityLabel: {
+    id: 'app.containers.Admin.projects.all.new.Projects.Filters.Visibility.label',
+    defaultMessage: 'Visibility',
+  },
+  visibilityPublic: {
+    id: 'app.containers.Admin.projects.all.new.Projects.Filters.Visibility.public',
+    defaultMessage: 'Public',
+  },
+  visibilityGroups: {
+    id: 'app.containers.Admin.projects.all.new.Projects.Filters.Visibility.groups',
+    defaultMessage: 'Groups',
+  },
+  visibilityAdmins: {
+    id: 'app.containers.Admin.projects.all.new.Projects.Filters.Visibility.admins',
+    defaultMessage: 'Admins',
+  },
+  visibilityUnlisted: {
```

Comment:
We don't use 'listed' and 'unlisted' in the FE copy- this was changed recently based on feedback from Lydie. We now use the concept of 'discoverability', which can be 'public' or 'hidden'

<img width="740" height="330" alt="Captura de Tela 2025-07-22 às 11 39 59" src="https://github.com/user-attachments/assets/e96b4004-1e62-4bb1-b764-e9dba0adca02" />